hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/AbstractMultiTargetMessageTask.java

```diff
@@ -37,57 +37,54 @@
 
 public abstract class AbstractMultiTargetMessageTask<P> extends AbstractMessageTask<P> {
 
-    private static final int TRY_COUNT = 100;
-
     protected AbstractMultiTargetMessageTask(ClientMessage clientMessage, Node node, Connection connection) {
         super(clientMessage, node, connection);
     }
 
     @Override
     protected void processMessage() throws Throwable {
         ClientEndpoint endpoint = getEndpoint();
-        OperationFactory operationFactory = createOperationFactory();
-        Collection<Address> targets = getTargets();
+        Supplier<Operation> operationFactory = createOperationFactory();
```

Comment:
since the OperationFactory has been replaced by supplied, it makes sense to have the local variable 'operationFactory' renamed to 'operationSupplier' or something along those lines.